resources/usa_mechanicjob/classes/MechanicHelper_cl.lua

```diff
@@ -130,11 +129,20 @@ end
 MechanicHelper.repairVehicle = function(veh, repairCount, cb)
     local beforeRepairHealth = GetVehicleEngineHealth(veh)
     local success = false
+    
+    if repairCount >= MechanicHelper.LEVEL_3_RANK_THRESH then
+        calculatedRepairTime = 20000
+    elseif repairCount >= MechanicHelper.LEVEL_2_RANK_THRESH then
+        calculatedRepairTime = 40000
+    else 
```

Comment:
```suggestion
    else 
```

then remove this else statement since it won't be needed anymore